services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/common/AttachmentValidator.kt

```diff
@@ -68,7 +73,7 @@ object FileNameValidator {
             "LPT9"
         )
 
-    fun validateFileName(input: String?): String {
+    fun validFileName(input: String?): String {
```

Comment:
This returns the same file name that was the input, it's in no way more valid than the input, except it can't be null. I prefer the original name for the function.

Reply:
I somewhat feel that using !! here and there is a bit of an anti-pattern. But ok, multipartfile comes from Java and there's not much we can do about the nullability.

Reply:
Changed as suggested (almost).